models/marts/core/marketing/dim_customers.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+
+models:
+  - name: dim_customers
+    tests:
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('stg_ecommerce__customers')
+    description: "Table contains metrics for each customer. This test will compare row counts with staging customer table to ensure that dim_customers contains unique row for each Customer"
```

Comment:
this description is at the same indentation level as the name so you can see it applies to the whole model; including details about the test is useful but it feels a bit odd in terms of organization